lib/libraries/default/table.js

```diff
@@ -0,0 +1,92 @@
+var each = require('../../utils/each');
+
+var defaults = {
+  height: undefined,
+  width: undefined,
+  stickyHeader: true,
+  stickyFooter: false
+};
+
+module.exports = {
+  render: function(){
+    var dataset = this.data(),
+        el = this.el(),
+        height = (this.height() || defaults.height) - this.el().offsetHeight,
+        theme = this.theme(),
+        width = this.width() || defaults.width;
+
+    var html = '',
+        colAligns = new Array(dataset[0].length),
+        colWidths = new Array(dataset[0].length),
+        fixedHeader;
+
+    // Calculate max column widths
+    each(dataset, function(row){
+      each(row, function(cell, i){
+        if (!colWidths[i]) {
+          colWidths[i] = 0;
+        }
+        colAligns[i] = (typeof cell === 'number') ? 'right' : 'left';
+        colWidths[i] = (String(cell).length > colWidths[i]) ? String(cell).length : colWidths[i];
+      });
+    });
+
+    // Open wrapper
+    html += '<div class="' + theme + '-table" style="height: '+(height ? height+'px' : 'auto')+'; width: '+(width ? width+'px' : 'auto')+';">';
```

Comment:
This is probably ok for the short term, but I feel string concatenation can get tough to maintain over time. We might want to look into something really lightweight, like `doT.js` (120 LOC) or something similar. http://olado.github.io/doT/index.html